exotica_core/package.xml

```diff
@@ -28,7 +28,7 @@
   <depend>eigen_conversions</depend>
   <depend>tf_conversions</depend>
   <depend>tinyxml2</depend>
-  <depend>libzmq3-dev</depend>
+  <depend>zeromq3</depend>
```

Comment:
Is this the correct key or does it change behaviour?

Reply:
I just saw your comment:
> Edit: There are two rosdep keys for ZeroMQ: libzmq3-dev and zeromq3. The package actually got installed.

So, can we revert to the other one?

Reply:
I opened an issue at rosdistro (https://github.com/ros/rosdistro/issues/21879) and wanted to get feedback on this. IMHO, `zeromq3` is the more generic name of the library because `libzmq3-dev` is already the name of the Debian/Ubuntu package. In any case, they both result in installing the package `libzmq3-dev` on Debian/Ubuntu. I was just hoping that this issue gets addressed in rosdistro before merging this.

Reply:
I am happy to stay with the more generic one then. Thanks for following this up upstream